_includes/header.html

```diff
@@ -0,0 +1,16 @@
+<header class="site-header">
+    <div class="ui container">
+        <div class="ui large secondary pointing menu">
+            <div class="left item">
+                <a href="{{ site.baseurl }}"><img class="ui tiny image" src="{{ site.baseurl }}/img/logo-marianne.gif"></a>
+            </div>
+            <a class="active item" href="#">API Guide</a>
+            <a class="item" href="#mapid">Essayer</a>
+            <div class="right item">
+                <a class="ui blue button menu" href="{{ site.documention }}">Documentation</a>
```

Comment:
`site.documention` -> `site.documentation`

Also this button seems redundant with the one in green?